### PR TITLE
docs(chore): proofread comments

### PR DESCRIPTION
**Pull request by aimeeu**
Proofread and update comments 

* updated links to existing CDSH docs
* standard terms 

In the CDaaS docs, I can display the contents values.yml using a custom shortcode. That's probably a better approach than duplicating the config settings in markdown.

Most, if not all, of the README is great content for the advanced install page (which you will have the pleasure of reviewing when the PR is ready). 

Partial: [CDAAS-2581]

[CDAAS-2581]: https://armory.atlassian.net/browse/CDAAS-2581?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -48,8 +48,8 @@ image:
 #  - name: regcred
 imagePullSecrets: []
```
Comment (aimeeu) [inline values.yaml:49]: Justin - what is `imagePullSecrets` for?

File: values.yaml [hunk 2]
```diff
@@ -69,30 +69,30 @@ proxy:
...
-# The created service account w/ cluster role binding is applied to the agent.
-# The agent then registers its self as a deployable Kubernetes target from within Armory Cloud.
+# When Kubernetes cluster account mode is enabled, installation creates a ServiceAccount, ClusterRole, and ClusterRoleBinding.
+# The created ServiceAccount with ClusterRoleBinding is applied to the RNA.
+# The RNA then registers itself as a deployable Kubernetes target from within CD-as-a-Service.
 #
-# When Kubernetes cluster account mode is disabled, the agent only allows you to make network calls to networked resources.
-# You will have to configure Kubernetes Accounts in the Armory Cloud Console to have deployable Kubernetes targets.
+# When Kubernetes cluster account mode is disabled, the RNA only allows you to make network calls to networked resources.
+# You have to configure Kubernetes accounts in the CD-as-a-Service Console to have deployable Kubernetes targets.
```
Comment (aimeeu) [inline values.yaml]:
```suggestion
```

Justin - is L77 still true?

File: values.yaml (hunk 1 above)
Comment (fieldju) [reply values.yaml:49]: Its a config option that Kubernetes exposed that enables using containers that are hosted in private registries:

https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod

File: values.yaml (hunk 2 above)
Comment (fieldju) [reply values.yaml]: No it't as of now, we removed the ability to upload kubeconfig files manually to CD-as-a-Service

**Review by fieldju — APPROVED** [review]

File: values.yaml (hunk 1 above)
Comment (fieldju) [reply values.yaml:49]: image

Comment (fieldju) [reply values.yaml:49]: Lastly this link might be most relevant and is how I was introduced to the concept of imagePullSecrets:
https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/

**Pull request merged**